projects/Allwinner/patches/linux/0061-arm64-dts-allwinner-Fix-SD-card-CD-GPIO-for-SOPine-s.patch

```diff
@@ -38,18 +38,6 @@ index 437ffe3628a5..e79ce49e7e6a 100644
 +&mmc0 {
 +	cd-gpios = <&pio 5 6 GPIO_ACTIVE_LOW>; /* PF6 push-push switch */
 +};
-diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-sopine.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-a64-sopine.dtsi
```

Comment:
You should remove whole patch. Partial removal probably works but causes two mmc0 nodes.

Reply:
Done. - full,patch dropped.